### PR TITLE
Installation error CPM-24

### PR DESCRIPTION
**Pull request by terryyoon-pdftron**
https://linear.app/pdftron/issue/CPM-24/[support-31589]-yarn-installation-issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +36,6 @@ The release can be found here: https://github.com/PDFTron/pdftron-react-native/r
 
 ## Installation
 
-0. If using yarn, do: `yarn global add react-native-cli`
```
Comment (JamieDass) [inline README.md:39]: Is this no longer needed?

Comment (terryyoon-pdftron) [reply README.md:39]: yeah, it works either way (added or not added)

File: README.md [hunk 2]
```diff
@@ -48,19 +46,19 @@ The release can be found here: https://github.com/PDFTron/pdftron-react-native/r
 
      ```shell
      yarn add github:PDFTron/pdftron-react-native
-     yarn add @react-native-community/cli --dev
-     yarn add @react-native-community/cli-platform-android --dev
-     yarn add @react-native-community/cli-platform-ios --dev
+     yarn add @react-native-community/cli@7.0.1 --dev
```
Comment (sgong-pdftron) [inline README.md:49]: why a specific version is needed? how are we going to maintain this?

**Review by sgong-pdftron — APPROVED** [review]
please make sure setup process is smooth with these changes

**Review by JamieDass — APPROVED** [review]
Same comment as Shirley, looks good as long as we're sure that this new process (omitting CLI packages) is well-tested and will work for all new installers.

**Pull request merged**